src/main/scala/viper/silver/parser/Translator.scala

```diff
@@ -265,6 +265,32 @@ case class Translator(program: PProgram) {
     }
   }
 
+  def fieldAssignStmt(errorNode: PNode, targets: Seq[PAssignTarget], assign: Seq[LocalVar] => Stmt): Stmt = {
+    val tTargets = targets map exp
+    val ts = tTargets.zipWithIndex.map {
+      case (lv: LocalVar, _) => (None, lv)
+      case (fa: FieldAccess, i) => {
+        val rcv = LocalVar(s"__silver_rcv_$i", fa.typ)(fa.rcv.pos)
```

Comment:
Maybe just __rcv_$i or something like that? I guess there's a chance one of these might show up in an error message somehow (?), so I'd avoid the "silver" part.

Also, here or below where you construct the Seqn, a comment explaining what these tuples are would be nice.